public/output/output.js

```diff
@@ -159,77 +198,71 @@ function addUsers() {
   return(output);
 }
 
-function generateUser() { // this function randomly generates a current user
-  let randomUserId = random(Object.keys(users));
-  let user = users[randomUserId]; // get the user
-
-  let username = user.username;
-  user.myTurn = true;
-
-  text(username, 100, 75); // update X&Y vals to wherever we put the username text
+// put next user's username at the top!
+function displayUser() {
+  push();
+  fill('magenta');
+  textAlign(CENTER);
+  textSize(120);
+  text(turnUser.username, 0, 0, canvasWidth, 100);
+  pop();
 }
 
-function generateImage() { // this function generates a random image (burger, fries, etc) on screen
-   // random number 0 to 4, where burger = 0, other images = 1-4
-  let randomImage = random(images);
-
-  // put the image on screen
-  image(randomImage, 100, 125); // update X&Y vals to be inside defined area
-
-  if (randomImage == 0) {             // burger
-    expectingFlip = true;
-    generateCountdown();
-  } else {                            // anything else
-    expectingShake = true;
-    shakeEvent();
-  }
+// put the image on screen
+function displayImage() {
+  push();
+  translate(canvasWidth / 2, canvasHeight / 2);
+  image(turnImage, -200, -200, 400, 400);
+  pop();
 }
 
-function generateCountdown() {  // this function generates a 3 sec countdown timer under the image
-  let seconds = second();
-  let timer = seconds + 3;
-  const timeoutText = '&#x1F525;&#x1F525;&#x1F525;' + 'YO BURGER GOT COOKED! ' + '&#x1F525;&#x1F525;&#x1F525;'
-
-  text("Countdown: ", 330, 75);
-  text(timer - seconds, 400, 75); // update X&Y vals to be inside defined area
-
-  if (frameCount % 60 == 0 && timer > 0) { // if the frameCount is divisible by 60, then a second has passed. it will stop at 0
-    timer--;
-  }
-  if (timer == 0) {
-    //text(timoutText, 450, 75);
-    if(expectingShake) {
-      shakeEvent();
-      expectingShake = false;
-    } else {
-      flipEvent();
-      expectingFlip = false;
-    }
+// this function generates a 3 sec countdown timer under the image
+function displayCountdown(timeLeft) {
+  const timeoutText = 'YO BURGER GOT COOKED!';
```

Comment:
Add flame emojis? &#x1F525;

Reply:
LOL I wasn't sure what those were but they weren't rendering for me. I'll try to put back later!